src/styles/blocks/card.css

```diff
@@ -0,0 +1,89 @@
+.card {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
```

Comment:
It is a great practice to create variables for the colors/fonts you use in your code. 
Imagine the page was redesigned - what will you do? You will need to fix all the colors/fonts manually. But if you use a variable - you can just change it in a single place